cosmic_ray/modules.py

```diff
@@ -35,6 +36,11 @@ def find_modules(name, excludes=None):
 
     """
     excludes = excludes or []
+    # helps when using bash completion
+    if os.path.exists(name):
+        if name.endswith('.py'):
+            name = name[:-3]
+        name = name.replace('/', '.')
```

Comment:
I would really prefer this logic to be in cli.py. This feels more like a "creature comfort" for command line users than it does a part of the `find_modules()` algorithm.